mkosi/resources/man/mkosi.1.md

```diff
@@ -204,80 +204,67 @@ The following command line verbs are known:
 
 ## Commandline-only Options
 
-Those settings cannot be configured in the configuration files.
+The following settings cannot be configured in the configuration files:
+
+`--help`, `-h`
+:   Show brief usage information.
 
 `--force`, `-f`
-:   Replace the output file if it already exists, when building an
-    image. By default when building an image and an output artifact
-    already exists `mkosi` will refuse operation. Specify this option
-    once to delete all build artifacts from a previous run before
-    re-building the image. If incremental builds are enabled,
-    specifying this option twice will ensure the intermediary
-    cache files are removed, too, before the re-build is initiated. If a
-    package cache is used (also see the **Files** section below),
-    specifying this option thrice will ensure the package cache is
-    removed too, before the re-build is initiated. For the `clean`
-    operation this option has a slightly different effect: by default
-    the verb will only remove build artifacts from a previous run, when
-    specified once the incremental cache files are deleted too, and when
-    specified twice the package cache is also removed.
+:   Replace the output file if it already exists, by default `mkosi`refuses to overwrite an existing image.
+    This option can be specified once, twice or trice to change the scope of artifact removals.
```

Comment:
Typo
```suggestion
    This option can be specified once, twice or thrice to change the scope of artifact removals.
```
but if we're touching this we should probably use something less poetic
```suggestion
    This option can be specified once, twice or three times to change the scope of artifact removals.
```
Same below.